x-pack/plugins/task_manager/server/lib/create_managed_configuration.ts

```diff
@@ -173,43 +193,56 @@ function countErrors(errors$: Observable<Error>, countInterval: number): Observa
           getMsearchStatusCode(e) === 503 ||
           getBulkUpdateStatusCode(e) === 429 ||
           getBulkUpdateStatusCode(e) === 500 ||
-          getBulkUpdateStatusCode(e) === 503
+          getBulkUpdateStatusCode(e) === 503 ||
+          isClusterBlockException(e)
       )
     )
   ).pipe(
     // When tag is "flush", reset the error counter
     // Otherwise increment the error counter
-    mergeScan(({ count }, next) => {
+    mergeScan(({ count, isBlockException }, next) => {
       return next === FLUSH_MARKER
-        ? of(emitErrorCount(count), resetErrorCount())
-        : of(incementErrorCount(count));
-    }, emitErrorCount(0)),
+        ? of(emitErrorCount(count, isBlockException), resetErrorCount())
+        : of(incrementOrEmitErrorCount(count, isClusterBlockException(next as Error)));
+    }, emitErrorCount(0, false)),
     filter(isEmitEvent),
-    map(({ count }) => count)
+    map(({ count, isBlockException }) => {
+      return { count, isBlockException };
+    })
   );
 }
 
-function emitErrorCount(count: number) {
+function emitErrorCount(count: number, isBlockException: boolean) {
   return {
     tag: 'emit',
+    isBlockException,
     count,
   };
 }
 
-function isEmitEvent(event: { tag: string; count: number }) {
+function isEmitEvent(event: { tag: string; count: number; isBlockException: boolean }) {
   return event.tag === 'emit';
 }
 
-function incementErrorCount(count: number) {
+function incrementOrEmitErrorCount(count: number, isBlockException: boolean) {
```

Comment:
I want to emit the error event as soon as possible in case of ClusterBlockException